app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, file_to_opening, path = command.split()
```

Comment:
path is not a descriptive name in this case

Reply:
Fixed